package.json

```diff
@@ -4,7 +4,7 @@
   "description": "common javascript utils that can be required selectively that assume es5+",
   "main": "index.js",
   "scripts": {
-    "test": "lab -c -t 100 -a code",
+    "test": "lab --leaks -c -t 100 -a code",
```

Comment:
[Tests can't be ran without `--leaks`](https://github.com/hapijs/lab/issues/581). I can remove this once I'm done with most items.